codegen_to_sdk_config.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Lets please put this file somewhere under the eng folder instead of in the root of the repo.